contents/docs/experiments/holdouts.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Holdouts
+---
+
+Holdouts let you create persistent groups of users who can be excluded from multiple experiments or all experiments.
+
+To create a new holdout, navigate to "Experiments" > "Holdout groups" and click "New holdout". Enter a descriptive name for your holdout, fill out the optional description to help you remember its purpose, and set the percentage of users you want to include in the holdout. We recommend between 1% and 10%; the holdout should be large enough to be statistically significant, but small enough as to not significantly reduce the pool of users available for experiments.
+
+Once created, you can assign the holdout when creating a new experiment, or while editing the experiment while it's in a draft state. The holdout becomes locked once any experiment using it is launched to prevent data inconsistencies that could affect results.
```

Comment:
```suggestion
Once created, you can assign the holdout to new or existing experiments. The holdout becomes locked once any experiment using it is launched and you will be unable to modify it. This is to prevent data inconsistencies that could affect results.
```